### ISSUE TITLE
any possibility to set color of input to white

### ISSUE THREAD

**Issue by bmatusiak** [taffyko/LCNiceChat#8]
the blue over blue is hard to see.

any possibility to set color of text input to white?

also would be nice to have a darker background behind chat box when active,  and a darker background behind new messages until fadeaway

**Comment by bmatusiak** [comment]
this changes text input to white.

```c#
[HarmonyPatch(typeof(HUDManager), "Update")]
[HarmonyPostfix]
private static void HUDUpdate(ref TMP_InputField ___chatTextField, ref HUDElement ___Chat)
{
    if (___chatTextField.isActiveAndEnabled) {
        if (___Chat.targetAlpha == 1.0f) {
            ___chatTextField.placeholder.color = Color.white;
        } else {
            ___chatTextField.placeholder.color = Color.blue;
        }
        ___chatTextField.textComponent.color = Color.white;
    }
}
```

**Comment by taffyko** [comment]
hi there! — i'm happy to say that this feature was just released in v1.2.5!
(the text color defaults to white, and can be configured)
